modules/ROOT/pages/6.0-release-notes-apis.adoc

```diff
@@ -0,0 +1,209 @@
+
+The flag returns `true` when an image being uploaded is removed after failing to upload.
+
+[[fakeclipboard]]
+==== `FakeClipboard`
+
+The new `FakeClipboard` API provides a means for storing state that can be set and accessed by plugins and the core editor.
+
+`FakeClipboard` is implemented on the {productname} global, allowing anything to write to it and read from it.
+
+[[resource-unload]]
+==== `Resource.unload`
+
+The new `Resource.unload` API makes it possible to unload resources in the resource loader.
+
+This allows tests, such as `ImageEmojiTest`, to remove existing databases before loading their own.
+
+[[setdata]]
+==== `setData` method in dialog API
+
+The dialog API’s `setData` method now uses a deep merge algorithm.
+
+Consequently, it now supports partial nested objects.
+
+[[promise]]
+==== success or failure callbacks replaced by `Promise` in multiple APIs
+
+* Previously, the AddOnManager, PluginManager, ScriptLoader, StyleSheetLoader, and ThemeManager APIs report resource loading status with `success` or `failure` callbacks.
++
+These APIs now, instead, return a `Promise`, with a value of either `resolve` or `reject`. Also, `EditorUpload`, which already returns a `Promise`, had callbacks removed.
+
+[[changed-apis]]
+=== Changed APIs
+
+[[closebutton]]
+==== `closeButton` and `timeout`
+
+* Previously, using `closeButton` to disable a _Notification’s_ close button required a `timeout` value to also be set.
++
+In {productname} 6, `closeButton` shows or hides the close button regardless of a `timeout` value being set or not set.
++
+Equivalently, setting a `timeout` value sets how long the notification presents before automatically closing, regardless of whether a close button is displayed or not.
+
+[[disabled]]
+==== `disabled` functions renamed in all {productname} APIs
+
+Previously, multiple {productname} APIs and UI classes used `disable` or `disabled` in their function or property names.
+
+In {productname} 6, these function names have been changed and configuration changes are required.
+
+. `isDisabled()` functions are now `isEnabled() functions.
+. The `enable()` and `disable()` functions are now the single `setEnabled(<state>)` function.
+. The `disable(<name>)` and `enable(<name>)` functions are now the single `setEnabled(<name>, <state>)` function in the Dialog APIs.
+
+As well, the `.disabled` property is now the `.enabled` property in all {productname} UI classes.
+
+[[dom-domutils]]
+==== `dom.DOMUtils`
+
+The `toHex` function has been removed from `dom.DOMUtils`.
+
+[[domparser]]
+==== `DomParser`
+
+The `filterNode` method has been removed from the `DomParser` API.
+
+As well, the `DomParser` API no longer uses a custom parser internally and instead uses the https://developer.mozilla.org/en-US/docs/Web/API/DOMParser[native `DOMParser` API].
+
+[[editor-setcontent]]
+==== `editor.SetContent`
+
+The `editor.SetContent` event’s `content` property has been deprecated.
```

Comment:
@lnewson Please check? Reads weird to me

Reply:
Almost correct, but yeah is a little "weird". I think the confusion is `editor.SetContent` isn't property on the `editor` API. Maybe something like this?
```suggestion
The `SetContent` editor event’s `content` property has been deprecated.
```